### ISSUE TITLE
OnboardingInfoViewController pushed twice on second next tap 

### ISSUE THREAD

**Issue by mozead1996** [corona-warn-app/cwa-app-ios#2203]
## Avoid duplicates

* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is not already reported in another issue

## Technical details

- Device name: Iphone 8 
- iOS Version: iOS 14.4
- App Version: release/2.0.x

## Describe the bug

A screen pushed twice in OnboardingInfoViewController's navigaitonController stack when click next button  

## Steps to reproduce the issue

1. Open app for first time 
2. Click on 'Let's get started'
3. Click on  'next'
4. Click on 'next' again 

## Expected behaviour 
 Expected behavior is that screen that contains "If you are diagnosed with corona virus " Header pushed only once  
## Possible Fix
I see the issue that completion handler called twice in switch statement in onBoardingViewController 
I will submit a PR 
## Additional context

---
Internal Tracking ID: [EXPOSUREAPP-5771](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5771)

**Comment by Ein-Tim** [comment]
Confirm this bug. Also visible on the 1.15.x branch AND 1.14.x branch (but it's not reproducible on a real iPhone 6s with 1.14.4 installed.

**Comment by mozead1996** [comment]
I submitted a PR with GIF representing the fix  @Ein-Tim

**Comment by Ein-Tim** [comment]
*Disclaimer:
I'm not a community manager or developer.*

@mozead1996 
I can confirm that the fix is working. 👍

**Comment by dsarkar** [comment]
---
Internal Tracking ID: [EXPOSUREAPP-5771](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5771)

**Comment by Ein-Tim** [comment]
#2225 was merged into the `Release 1.15.x` branch and I can confirm that the fix works. @dsarkar Please re-label to `Fix 1.15`.
@mozead1996 I think this issue can be closed now, or?
